Neos.Media/Classes/Command/MediaCommandController.php

```diff
@@ -127,6 +128,56 @@ public function importResourcesCommand($simulate = false)
     }
 
     /**
+     * Remove unused assets
+     *
+     * This command iterates over all existing assets, checks their usage count
+     * and lists the assets which are not reported as used by any AssetUsageStrategies.
+     * The unused assets can than be removed.
+     *
+     * @return void
+     */
+    public function removeUnusedCommand()
+    {
+        $iterator = $this->assetRepository->findAllIterator();
+        $assetCount = $this->assetRepository->countAll();
+        $unusedAssets = [];
+        $unusedAssetInfo = [];
+        $unusedAssetCount = 0;
+
+        $this->outputLine('<b>Searching for unused assets.</b>');
+
+        $this->output->progressStart($assetCount);
+        /** @var AssetInterface $asset */
+        foreach ($this->assetRepository->iterate($iterator) as $asset) {
+            if ($asset->getUsageCount() === 0) {
+                $unusedAssets[] = $asset;
+                $unusedAssetInfo[] = sprintf('- %s (%s)', $asset->getIdentifier(), $asset->getResource()->getFilename());
+                $unusedAssetCount++;
+            }
+            $this->output->progressAdvance(1);
+        }
+
+        $this->outputLine(PHP_EOL . 'Found the following unused assets: ' . PHP_EOL . implode(PHP_EOL, $unusedAssetInfo));
+
+        $this->output->outputLine(sprintf('Found <b>%s</b> unused assets.', $unusedAssetCount));
```

Comment:
would make more sense to output something like Found 3 unused assets: ... list assets

instead of how it is currently where it says Found the following unused assets: Found 0 unused assets.

And then just write "no unused assets found" if none are found.